src/main/java/com/google/sps/algorithms/MatchQuery.java

```diff
@@ -3,12 +3,14 @@
 import com.google.sps.data.MatchRequest;
 import com.google.sps.data.User;
 import com.google.sps.data.NonPersistentUserRepository;
+import com.google.sps.data.MatchRepository;
 import java.util.ArrayList;
 import java.util.Collection;
 
 public final class MatchQuery {
 
-  public Collection<User> query(MatchRequest request) {
+  public Collection<User> query(MatchRequest request, Collection<User> userSavedMatches) {
```

Comment:
maybe add some comments to describe how to use this method (e.g. what users get returned for a given request and userSavedMatched?)

Reply:
Done.